wallet/src/de/schildbach/wallet/util/WalletExtensions.kt

```diff
@@ -0,0 +1,11 @@
+package de.schildbach.wallet.util
+
+import de.schildbach.wallet.Constants
+import org.bitcoinj.core.Coin
+import org.bitcoinj.wallet.Wallet
+
+fun Wallet.canAffordIdentityCreation(): Boolean {
+    val walletBalance: Coin = getBalance(Wallet.BalanceType.ESTIMATED)
+    return (walletBalance.isGreaterThan(Constants.DASH_PAY_FEE)
+            || walletBalance == Constants.DASH_PAY_FEE)
+}
```

Comment:
My question is not related to this PR, but what happens when the user only has 0.01 DASH and they try to register.  Will it succeed or fail?  The reason for the question is that there is a fee that is required to send the transaction.  Though, in the case of sending all the value, then the fee may be taken out of the 0.01.  What if the balance is 0.01 - something less than the fee?  We can have QA figure this out and make a new ticket if there is a problem.

Reply:
@HashEngineering when I sent 0.01 to the new wallet and try to create an Identity it fails due to insufficient funds exception...